### PR TITLE
cleaned up configured vendors for provide plugin

### PR DESCRIPTION
**Pull request by thescientist13**
Please make sure the following criteria is meet before submitting your pull request.

1. <strong>PR meets the  [Contributing guidelines](./CONTRIBUTING.md)</strong>
1. Fork the repository and create your branch from <i>master</i>.
1. Ensure the test suite passes (<i>yarn run test</i>).
1. Make sure your code lints (<i> yarn run lint </i>).

Changes Made
1. This removes superfluous configuration from the `ProvidePlugin` configuration and verified the app still ran as expected without them.  
1. Additionally fixes the PR template and adds a couple more prompts

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -2,10 +2,16 @@
 We love PRs and appreciate any help you can offer.
 -->
 
+## PR Checlist
```
Comment (thisgeek) [inline .github/PULL_REQUEST_TEMPLATE.md]: @thescientist13 Did you mean to include these changes along with the ProvidePlugin fixes? They look unrelated.

Comment (thescientist13) [inline .github/PULL_REQUEST_TEMPLATE.md]: I did, but I probably shouldn't have.   I will put into a new PR

**Comment by thescientist13** [comment]
@thisgeek 
Rolled back the changes and opened an issue #21 and PR for it (#22)

**Comment by thescientist13** [comment]
looks like #19 essentially resolved this.  I expect #14 will address the need of making sure this is add to _.gitignore_
